packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -107,12 +107,12 @@ export default class PostsRepo extends AbstractRepo<DbPost> {
         SELECT
           "key",
           (ARRAY_AGG(
-            "displayName" ORDER BY COALESCE("karma", 0) DESC)
+            "displayName" ORDER BY "createdAt" ASC)
```

Comment:
Should this be ASC or DESC? I would have assumed more recent would be shown first. @agnesstenlund ?

Reply:
Answered on slack: oldest first